source/_components/fan.xiaomi_miio.markdown

```diff
@@ -194,7 +272,7 @@ name:
   type: string
   default: Xiaomi Air Purifier
 model:
-  description: The model of your miio fan. Valid values are `zhimi.airpurifier.v6`, `zhimi.airpurifier.v3`, `zhimi.humidifier.v1` and `zhimi.humidifier.ca1`. This setting can be used to bypass the device model detection and is recommended if your device isn't always available.
+  description: The model of your miio fan. Valid values are `zhimi.airpurifier.v1`, `zhimi.airpurifier.v2`, `zhimi.airpurifier.v3`, `zhimi.airpurifier.v5`, `zhimi.airpurifier.v6`, `zhimi.airpurifier.v7`, `zhimi.airpurifier.m1`, `zhimi.airpurifier.m2`, `zhimi.airpurifier.ma1`, `zhimi.airpurifier.ma2`, `zhimi.airpurifier.sa1`, `zhimi.airpurifier.sa2`, `zhimi.airpurifier.mc1`, `zhimi.humidifier.v1`, `zhimi.humidifier.ca1` and `zhimi.airfresh.va2`. This setting can be used to bypass the device model detection and is recommended if your device isn't always available.
```

Comment:
"See table above for valid values" or similar, avoids the need to keep this in sync :-) Feel free to merge if you like to keep it as it is!

Reply:
Updated. I'm not allowed to merge a unapproved PR.